app/models/rankinglist.rb

```diff
@@ -0,0 +1,3 @@
+class Rankinglist < Event
+  validates :discipline, :deadline, :startdate, :enddate, :player_type, presence: false
```

Comment:
Why does the Rankinglist not want a discipline or a player_type? Doesn't Event already check for discipline's presence?

Reply:
Btw. tests testing whether it is possible to actually create a Rankinglist would be really nice.